### DISCUSSION TITLE
Couldn't run simulating_long_experiments

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#2546] [Q&A]
I try to run simulating-long-experiments.ipynb(https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb)

However, an error occurred.
![에러](https://user-images.githubusercontent.com/113442395/207237143-95fedfbc-6857-4ca8-b766-91ecbc887b62.png)

My pybamm version is 22.11.
How can I solve this error?

Thank you.

**Comment by brosaplanella** [comment]
How have you defined your `parameter_values`?

**Reply by YouHyin** [reply]
parameter_values = pybamm.ParameterValues("Mohtat2020")
parameter_values.update({"SEI kinetic rate constant [m.s-1]": 1e-14})
spm = pybamm.lithium_ion.SPM({"SEI": "ec reaction limited"})


# Calculate stoichiometries at 100% SOC
parameter_values.set_initial_stoichiometries(1);

**Reply by YouHyin** [reply]
It was set the same as the code posted in the example.

**Comment by valentinsulzer** [comment]
This functionality changed a little bit as a result of #2508 

If you installed as a developer then pull the latest changes and update
If you installed from pip then use the old notebook from the `v22.11` release https://github.com/pybamm-team/PyBaMM/blob/v22.11/examples/notebooks/simulating-long-experiments.ipynb
